src/SourceBuild/tarball/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/baselines/PoisonUsage.txt

```diff
@@ -554,775 +638,841 @@
     <Type>AssemblyAttribute</Type>
   </File>
-  <File Path="Microsoft.DotNet.SharedFramework.Sdk.x.y.z/tools/netx.y/System.Runtime.CompilerServices.Unsafe.dll">
+  <File Path="Microsoft.Net.Compilers.Toolset.x.y.z/tasks/netx.y/bincore/runtimes/win/lib/netx.y/System.Text.Encoding.CodePages.dll">
     <Type>AssemblyAttribute</Type>
   </File>
-  <File Path="Microsoft.DotNet.SharedFramework.Sdk.x.y.z/tools/netx.y/System.Text.Encodings.Web.dll">
+  <File Path="Microsoft.Net.Compilers.Toolset.x.y.z/tasks/netx.y/bincore/System.Collections.Immutable.dll">
     <Type>AssemblyAttribute</Type>
   </File>
-  <File Path="Microsoft.DotNet.SharedFramework.Sdk.x.y.z/tools/netx.y/System.Text.Encodings.Web.dll">
+  <File Path="Microsoft.Net.Compilers.Toolset.x.y.z/tasks/netx.y/bincore/System.Reflection.Metadata.dll">
     <Type>AssemblyAttribute</Type>
   </File>
-  <File Path="Microsoft.DotNet.SharedFramework.Sdk.x.y.z/tools/netx.y/System.Text.Json.dll">
+  <File Path="Microsoft.Net.Compilers.Toolset.x.y.z/tasks/netx.y/bincore/System.Text.Encoding.CodePages.dll">
     <Type>AssemblyAttribute</Type>
   </File>
-  <File Path="Microsoft.DotNet.SharedFramework.Sdk.x.y.z/tools/netx.y/System.Text.Json.dll">
+  <File Path="Microsoft.Net.Compilers.Toolset.x.y.z/tasks/netx.y/bincore/runtimes/win/lib/netx.y/System.Text.Encoding.CodePages.dll">
     <Type>AssemblyAttribute</Type>
   </File>
-  <File Path="Microsoft.DotNet.SignTool.x.y.z/lib/netx.y/Newtonsoft.Json.dll">
+  <File Path="Microsoft.Net.Compilers.Toolset.x.y.z/tasks/netx.y/bincore/System.Collections.Immutable.dll">
     <Type>AssemblyAttribute</Type>
   </File>
-  <File Path="Microsoft.DotNet.SignTool.x.y.z/lib/netx.y/Newtonsoft.Json.dll">
+  <File Path="Microsoft.Net.Compilers.Toolset.x.y.z/tasks/netx.y/bincore/System.Reflection.Metadata.dll">
     <Type>AssemblyAttribute</Type>
   </File>
-  <File Path="Microsoft.DotNet.SignTool.x.y.z/lib/netx.y/NuGet.Common.dll">
+  <File Path="Microsoft.Net.Compilers.Toolset.x.y.z/tasks/netx.y/bincore/System.Text.Encoding.CodePages.dll">
     <Type>AssemblyAttribute</Type>
   </File>
-  <File Path="Microsoft.DotNet.SignTool.x.y.z/lib/netx.y/NuGet.Common.dll">
+  <File Path="Microsoft.NET.ILLink.Tasks.x.y.z/tools/net472/System.Collections.Immutable.dll">
     <Type>AssemblyAttribute</Type>
   </File>
-  <File Path="Microsoft.DotNet.SignTool.x.y.z/lib/netx.y/NuGet.Configuration.dll">
+  <File Path="Microsoft.NET.ILLink.Tasks.x.y.z/tools/net472/System.Reflection.Metadata.dll">
     <Type>AssemblyAttribute</Type>
   </File>
-  <File Path="Microsoft.DotNet.SignTool.x.y.z/lib/netx.y/NuGet.Configuration.dll">
+  <File Path="Microsoft.NETCore.App.Host.x.y.z/runtimes/fedora.33-x64/native/coreclr_delegates.h">
+    <Type>Hash</Type>
+  </File>
+  <File Path="Microsoft.NETCore.App.Host.linux-x64.x.y.z/runtimes/linux-x64/native/coreclr_delegates.h">
+    <Type>Hash</Type>
+  </File>
+  <File Path="runtime.linux-x64.Microsoft.DotNet.ILCompiler.x.y.z/tools/libobjwriter.so">
```

Comment:
The portable rid be abstracted by https://github.com/dotnet/installer/blob/main/src/SourceBuild/tarball/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/BaselineHelper.cs#L95.  Without this the poison report is architecture specific.